github_data/methods.py

```diff
@@ -143,8 +144,8 @@ def get_new_contributors_open(df):
 
 def get_summary(df):
     """
-    Count the number of issues or pull requests by week selected from the data
-    frame with issues or pull requests. Count only issues or pull requests
+    Count the number of issues or pull requests by creation date selected from
+    the data frame with issues or pull requests. Count only issues or pull requests
```

Comment:
```suggestion
    the data frame with issues or pull requests. Count only issues or pull requests.
```
Arbitrary change to trigger a commit!